Ocelot.Downstream.Service.Instance2/Ocelot.Samples.OcelotConsulServiceDiscovery.Instance2.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
Multi target is a better choice, use TargetFrameworks, please check the new Ocelot project file